slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -61,6 +67,8 @@ trait OracleProfile extends JdbcProfile {
 
   override protected def computeCapabilities: Set[Capability] = (super.computeCapabilities
     - RelationalCapabilities.foreignKeyActions
+    - RelationalCapabilities.createIfNotExists
+    - RelationalCapabilities.dropIfExists
```

Comment:
@cvogt Thanks... I should change that and also update the others, only DerbyProfile should have these removed since it uses a client side implementation.